src/test/java/org/apache/sling/resourceresolver/impl/mapping/ResourceMapperImplTest.java

```diff
@@ -100,7 +104,10 @@ public void prepare() throws LoginException {
         ctx.registerInjectActivateService(new StringInterpolationProviderImpl());
 
         InMemoryResourceProvider resourceProvider = new InMemoryResourceProvider();
-        resourceProvider.putResource("/"); // root
+
+        if(!testName.getMethodName().contains("mapEmptyPathWithUnreadableRoot")) {
```

Comment:
This was done to recreate the scenario observed in the bug where no resource as "/" added.

Reply:
I am still interested in finding out how the unreadable root can take happen. Please describe the scenario in detail - either here, or in the Jira issue so that we can reproduce it in the Sling Starter. A content package that works with the Sling Starter would be ideal, but I can also work with "textual" descriptions.

If this becomes too complicated for this unit testing framework we can always add an integration test in https://github.com/apache/sling-org-apache-sling-launchpad-integration-tests .

But first, I want to understand the problem.